ros2_controllers_test_nodes/ros2_controllers_test_nodes/publisher_forward_position_controller.py

```diff
@@ -25,14 +25,14 @@ class PublisherForwardPosition(Node):
     def __init__(self):
         super().__init__("publisher_forward_position_controller")
         # Declare all parameters
-        self.declare_parameter("controller_name", "forward_position_controller")
+        self.declare_parameter("publish_topic", "/position_commands")
```

Comment:
I would rather go with empty and force users to set it. What do you think?

```suggestion
        self.declare_parameter("publish_topic", "")
```

Reply:
yes but if there are any tests relying on that those sh ould also be updated (in demos I guess)

Reply:
Demos are currently independent of this since they use old implementation in the `ros2_control_demos` repository.